src/code/uiCommands.js

```diff
@@ -388,6 +388,44 @@ export function zoomToOperation(operation) {
   }
 }
 
+export async function updateLocalOp(local, remote) {
+  const so = getSelectedOperation();
+  if (!local || !local.localchanged) {
+    // merge blockers and related portals
+    if (local) remote.mergeBlockers(local);
```

Comment:
is this line redundant with 393? we can't be !local and local

Reply:
it can be `local` and `!local.localchanged`